<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bhargavi" failures="1"
	tests="2" name="test1(failed)(failed)" time="0.029" errors="0"
	timestamp="21 Oct 2024 11:35:41 GMT">
	<testcase classname="TestNgListeners.com.RetryTestUsingTestNg"
		name="test2" time="0.011">
		<failure type="java.lang.AssertionError"
			message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at TestNgListeners.com.RetryTestUsingTestNg.test2(RetryTestUsingTestNg.java:19)
... Removed 27 stack frames]]>
		</failure>
	</testcase> <!-- test2 -->
	<testcase classname="TestNgListeners.com.RetryTestUsingTestNg"
		name="retrytst1" time="0.002" />
</testsuite> <!-- test1(failed)(failed) -->
